akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -98,15 +106,34 @@ class Http2ServerDemux extends GraphStage[BidiShape[Http2SubStream, FrameEvent,
         bufferedFrameOut.push(SettingsFrame(Nil)) // server side connection preface
       }
 
-      var incomingStreams = Map.empty[Int, SubStream]
+      var closedAfter: Option[Int] = None
```

Comment:
add a comment that this is for handling the "we should not handle streams later than the GOAWAY told us about with lastStreamId"